app/src/routes/v1/keycloak.js

```diff
@@ -0,0 +1,42 @@
+const keyCloak = require('express').Router();
+//const { param, validationResult } = require('express-validator');
+const Problem = require('api-problem');
+//const validator = require('validator');
+const log = require('npmlog');
+
+
+// fetches all the service clients for all acronyms in all KC realms
+keyCloak.get('/clients', [
+], async (req, res) => {
+  // requires user role "GETOK_ADMIN"
+  if (!req.user.jwt.realm_access.roles.includes('GETOK_ADMIN')) {
+    // mock for now
+    // do we need a GETOK_ADMIN permission check?
+    const permissionErr = false;
+    if (permissionErr) {
+      return res.status(403).json({
+        message: permissionErr
+      });
```

Comment:
You'll want to update this to be using the api-problem path instead of directly responding.